src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1249,6 +1263,14 @@ class ContentExplorer extends Component<Props, State> {
         this.setState({ currentOffset: newOffset }, this.refreshCollection);
     };
 
+    getViewMode = (): ViewMode => {
+        const { features }: Props = this.props;
+        const viewModePreference = this.store.getItem(localStoreViewMode);
+        return viewModePreference && isFeatureEnabled(features, 'contentExplorer.gridView.enabled')
```

Comment:
This may be a bit easier to read if we break the `isFeatureEnabled` result into its own variable.